std_msgs/Header header
# controller gains (position)
float64 kp
float64 kd
# controller gains (force)
float64 km_f
float64 kd_f
# desired position
float64 x
float64 y
# desired force
float64 z
#circle trajectory
bool circle_trj
float64 frequency
float64 radius
float64 center_x
float64 center_y
